board.js

```diff
@@ -0,0 +1,64 @@
+class	Board
```

Comment:
Name of the file of class object: Board.js

Reply:
Ok, I have now written all class files with capital letters